apex-squid/src/test/java/org/fundacionjala/enforce/sonarqube/apex/api/ApexGrammarClassOrInterfaceBodyDeclarationTest.java

```diff
@@ -21,22 +21,23 @@
  * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
  * THE SOFTWARE.
  */
-package org.fundacionjala.enforce.sonarqube.apex.api.grammar.body;
+package org.fundacionjala.enforce.sonarqube.apex.api;
 
-import static org.fundacionjala.enforce.sonarqube.apex.api.grammar.RuleKey.CLASS_OR_INTERDACE_BODY_DECLARATION;
+import com.sonar.sslr.api.Grammar;
 import org.junit.Test;
-import org.sonar.sslr.grammar.LexerlessGrammarBuilder;
 import static org.sonar.sslr.tests.Assertions.assertThat;
+import static org.fundacionjala.enforce.sonarqube.apex.api.grammar.RuleKey.CLASS_OR_INTERFACE_BODY_DECLARATION;
 
 public class ApexGrammarClassOrInterfaceBodyDeclarationTest {
```

Comment:
Lack comments on all unit tests, please add comments follow code conventions of java